server/server.js

```diff
@@ -1,14 +1,28 @@
 const path = require("path")
 const express = require("express");
+const axios = require("axios");
 const bodyParser = require("body-parser");
+const token = require("../config.js");
 
 const app = express();
 const PORT = 3000;
+const apiHost = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp';
 
 app.use(express.static(path.join(__dirname, "/../client/dist")));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({extended: true}));
 
+app.get('/reviews/:product_id/:sort', (req, res) => {
+  var {product_id, sort} = req.params;
+  var url = `${apiHost}/reviews/?product_id=${product_id}&sort=${sort}&page=1&count=2`;
+  axios.get(url, {
+    headers: {
+      Authorization: token.token
+    }
+  })
+  .then(data => res.send(data.data))
```

Comment:
you can de-structure the data parameter, to just write data in the function body

i.e.
{data} => res.send(data)